mslib/msui/mpl_pathinteractor.py

```diff
@@ -441,6 +436,7 @@ def redraw_path(self, vertices=None, waypoints_model_data=None):
         output. Otherwise the vertex array obtained from the path patch
         will be used.
         """
+        assert False
```

Comment:
We can not rely on asserts in production because

for performance reasons, assert statements can be disabled

Reply:
I put that there to identify dead code and forgot about it. Nice catch.

Then this whole method is dead code.